x-pack/plugins/security_solution/public/overview/containers/overview_cti_links/use_request_event_counts.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { useMemo } from 'react';
+import { i18n } from '@kbn/i18n';
+import { convertToBuildEsQuery } from '../../../common/lib/keury';
+import { esQuery } from '../../../../../../../src/plugins/data/public';
+import { MatrixHistogramType } from '../../../../common/search_strategy';
+import { EVENT_DATASET } from '../../../../common/cti/constants';
+import { useMatrixHistogram } from '../../../common/containers/matrix_histogram';
+import { useKibana } from '../../../common/lib/kibana';
+
+export const useRequestEventCounts = (to: string, from: string) => {
+  const { uiSettings } = useKibana().services;
+
+  const matrixHistogramRequest = useMemo(() => {
+    return {
+      endDate: to,
+      errorMessage: i18n.translate('xpack.securitySolution.overview.errorFetchingEvents', {
+        defaultMessage: 'Error fetching events',
+      }),
+      filterQuery: convertToBuildEsQuery({
+        config: esQuery.getEsQueryConfig(uiSettings),
+        indexPattern: {
+          fields: [
+            {
+              name: 'event.kind',
+              aggregatable: true,
+              searchable: true,
+              type: 'string',
+              esTypes: ['keyword'],
+            },
+          ],
+          title: 'filebeat-*',
+        },
+        queries: [{ query: 'event.type:indicator', language: 'kuery' }],
+        filters: [],
+      }),
+      histogramType: MatrixHistogramType.events,
+      indexNames: ['filebeat-*'],
```

Comment:
Similar to [investigation enrichment](https://github.com/elastic/security-team/issues/1002#issuecomment-867095361), we should discuss whether to hardcode this to filebeat or to respect the user's configured datasources on the overview page. An argument against hardcoding is that it discourages users from defining their own indicator indexes (outside of the `filebeat-*` pattern, at least).

Reply:
cool, since the discussion is already taking place at the linked thread I am not creating a new issue - I will assign this to a constant for now in the CTI constants file for better consistency.